base/float.jl

```diff
@@ -737,6 +738,10 @@ julia> 1.0 + eps()
 julia> 1.0 + eps()/2
 1.0
 ```
+
+For complex inputs `T`, `eps(T)` is defined as the distance bound to the nearest
+floating-point complex value, i.e. for z=a+ib and the closest floating-point
+complex value z̃=ã+ib̃=fl(a)+ifl(b), |z - z̃| ≤ ε/2.
```

Comment:
`fl(x)` is not a completely standard mathematical symbol (it is used in Trefethen's book and probably some other places), so we should probably state *where fl(x) is the closest floating-point value to a real x*.

We should probably also say |z - z̃| ≤ eps(z)/2.